packages/web-components/src/components/va-number-input/va-number-input.css

```diff
@@ -0,0 +1,5 @@
+@import '../va-text-input/va-text-input.css';
+
+input:invalid {
```

Comment:
@bkjohnson what I had found was these are the parameters that work with it: required, minlength, maxlength, min, max, type and pattern.
Since min and max is unique to the `va-number-input` I added it.
Here's an article going into more depth: https://funwithforms.com/posts/valid-invalid-pseudo-classes/